roles/openshift_excluder/README.md

```diff
@@ -1,50 +1,69 @@
 OpenShift Excluder
-================
+==================
 
 Manages the excluder packages which add yum and dnf exclusions ensuring that
-the packages we care about are not inadvertantly updated. See
+the packages we care about are not inadvertently updated. See
 https://github.com/openshift/origin/tree/master/contrib/excluder
 
 Requirements
 ------------
-openshift_facts
 
+None
 
-Facts
------
+Inventory Variables
+-------------------
 
-| Name                       | Default Value | Description                            |
------------------------------|---------------|----------------------------------------|
-| enable_docker_excluder     | enable_excluders | Enable docker excluder. If not set, the docker excluder is ignored. |
-| enable_openshift_excluder  | enable_excluders | Enable openshift excluder. If not set, the openshift excluder is ignored. |
-| enable_excluders           | None             | Enable all excluders
+| Name                                 | Default Value              | Description                            |
+---------------------------------------|----------------------------|----------------------------------------|
+| openshift_enable_excluders           | True                       | Enable all excluders                   |
+| openshift_enable_docker_excluder     | openshift_enable_excluders | Enable docker excluder. If not set, the docker excluder is ignored. |
+| openshift_enable_openshift_excluder  | openshift_enable_excluders | Enable openshift excluder. If not set, the openshift excluder is ignored. |
 
 Role Variables
 --------------
-None
```

Comment:
@mtnbikenc Anything not passed into the role is the role requiring global variables to be set.

So if I want to call this role outside of the installer / upgrader, I now have to populate my inventory to look like what this role is expecting.

This pattern is very analogous to global variables being used inside a function or class. If course it's easier, but it makes your API very difficult to work with by external consumers.

@sdodson Can we discuss this in the next openshift-ansible arch meeting please?  The pattern used here is really going against the intention of the "don't use global variables in roles" rule that we agreed to previously, so we clearly need more discussion and buy off from the team on it.